countryCodeToTimeZones.go

```diff
@@ -0,0 +1,2259 @@
+package libphonenumber
+
+import "strconv"
+
+const MAX_REGION_CODE_LENGTH = 7
+
+var CountryCodeToTimeZones = map[int][]string{
```

Comment:
Can we add a comment here as to what the structure of this mapping means semantically?